### ISSUE TITLE
Entry with dca field 'cfgTags' can´t be deleted

### ISSUE THREAD

**Issue by jaynoe** [codefog/tags-bundle#40]
I have the dca inputType 'cfgTags' in two custom tables and in the calendar events table - if i want to delete an entry in the backend via the delete function an error showed up:

`Uncaught PHP Exception TypeError: "json_decode(): Argument #1 ($json) must be of type string, null given" at /xxx/xxx/vendor/codefog/contao-haste/src/UndoManager.php line 56`

My dca declaration is like this:

```php
'tags' => array
        (
            'label' => &$GLOBALS['TL_LANG']['MSC']['tags'],
            'exclude' => true,
            'inputType' => 'cfgTags',
            'eval' => [
                'tagsManager' => 'hh_tags_manager', // Manager name, required
                'tagsCreate' => false, // Allow to create tags, optional (true by default)
                'hideList' => true, // Hide the list of tags; the input field will be still visible
                'tl_class' => 'clr'
            ]
        )
```

An entry in tl_undo is made - but when i checked it in the backend, another error came up:

`Uncaught PHP Exception RuntimeException: "System::import() failed because class "Util\Undo" is not a valid class name or does not exist." at /xxx/xxx/vendor/contao/core-bundle/src/Resources/contao/library/Contao/System.php line 195`

I´m using
- Contao 4.13.13 (tried versions till 4.13.10)
- tags-bundle 3.3.9

**Comment by qzminski** [comment]
Actually, this is a problem of the `codefog/contao-haste`. I have just released a new version 5.0.2. Could you update all the dependencies and see if it works?

**Comment by jaynoe** [comment]
Yes, now it works! Thanks!
